tools/isobuild/linker.js

```diff
@@ -241,9 +335,26 @@ var File = function (inputFile, module) {
   // produced by plugins)
   self.sourceHash = inputFile.hash || watch.sha1(self.source);
 
+  // The path of the source file, relative to the root directory of the
+  // package or application.
+  self.sourcePath = inputFile.sourcePath;
+
+  // Absolute module identifier to use when installing this file via
+  // meteorInstall.
+  self.installPath = inputFile.installPath || inputFile.sourcePath;
```

Comment:
When is `inputFile.installPath` falsey? Probably warrants a comment. (I haven't reviewed import-scanner.js yet)